libs/remix-simulator/src/vm-context.ts

```diff
@@ -24,31 +24,27 @@ class StateManagerCommonStorageDump extends StateManager {
     return super.putContractStorage(address, key, value)
   }
 
-  async dumpStorage (address) {
-    let trie
-    try {
-      trie = await this._getStorageTrie(address)
-    } catch (e) {
-      console.log(e)
-      throw e
-    }
-    return new Promise<StorageDump>((resolve, reject) => {
-      try {
-        const storage = {}
-        const stream = trie.createReadStream()
-        stream.on('data', (val) => {
-          const value = rlp.decode(val.value)
-          storage['0x' + val.key.toString('hex')] = {
-            key: this.keyHashes[val.key.toString('hex')],
-            value: '0x' + value.toString('hex')
-          }
+  async dumpStorage (address): Promise<StorageDump> {
+    return new Promise((resolve, reject) => {
+      this._getStorageTrie(address)
```

Comment:
Any specific reason to remove `await`

Reply:
I wanted to be as close as possible to the original `dumpStorage` https://github.com/ethereumjs/ethereumjs-monorepo/blob/501be8da11b07d138bb4497f8a18bdf0fa2d11d7/packages/vm/src/state/stateManager.ts#L519

Reply:
ok